Lib/test/test_unicode.py

```diff
@@ -1630,6 +1630,10 @@ def test_codecs_utf7(self):
         for c in set_o:
             self.assertEqual(c.encode('ascii').decode('utf7'), c)
 
+        with self.assertRaisesRegex(UnicodeDecodeError,
```

Comment:
Please add also a test for the non-strict error handler, e.g. "replace". `UTF7Test.test_errors` in `test_codecs.py` may be more appropriate place for this test.

Reply:
Thanks for the comment. I've added the test.

Reply:
The test in `test_unicode.py` is no longer needed, isn't it?

Reply:
The test in `test_codecs.py` doesn't check the error message.

Reply:
Then all correct.